jenkins-build-pack-api/Jenkinsfile

```diff
@@ -212,9 +203,9 @@ node() {
 					if(domain != "platform"){
```

Comment:
We don't need this if clause. Platform services endpoint info will be updated and so will be the API Docs.

Reply:
In cooperated the review comment. But that if clause is needed because , that block is for creating the subscription filters for the user services.